_includes/app/activerecord-basic-sample.rb

```diff
@@ -42,5 +42,5 @@ def change
 
 # Retrieve accounts and print out the balances
 Account.all.each do |acct|
-    puts "#{acct.id} #{acct.balance}"
+  puts "#{acct.id} #{acct.balance}"
```

Comment:
In Ruby you can't use string interpolation within single quotes (they're just completely literal strings), so this one has to be double quoted. Afaik (and a Google seems to confirm) there isn't a standard style for which (single or double) to use by default, unfortunately.